chapter4.Rmd

```diff
@@ -418,7 +412,7 @@ test_object("factor_survey_vector", eq_condition = "equal", undefined_msg = msg,
 test_object("male", undefined_msg = msg, incorrect_msg = msg)
 test_object("female", undefined_msg = msg, incorrect_msg = msg)
 test_output_contains("male > female", incorrect_msg = msg)
-success_msg("Phew, it seems that R is gender-neutral. Or maybe it just wants to stay out of these discussions ;-).")
+success_msg("Phew, it seems that R is not a sexist language.")
```

Comment:
I think I'd make this just a standard "Good job." `success_msg`.

Reply:
:+1:

Reply:
Updated via https://github.com/datacamp/courses-intro-to-r/commit/be438ff3ad1dd9bd9f9cbd761000a4a676bda0a9